qiskit_experiments/library/calibration/analysis/fine_drag_analysis.py

```diff
@@ -0,0 +1,21 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine DRAG calibration analysis."""
+
+from .fine_amplitude_analysis import FineAmplitudeAnalysis
+
+
+class FineDragAnalysis(FineAmplitudeAnalysis):
```

Comment:
Inheriting this from `FineAmplitudeAnalysis` looks bit strange. Can you rename it to something like `PingPongAnalysis`?

Reply:
Good point. See here: 81447ba and here 353b6a2 I went for `ErrorAmplificationAnalysis` which is more descriptive.